charts/minikube-values.yaml

```diff
@@ -13,6 +13,8 @@ ingress:
 global:
   gitlab:
     urlPrefix: /gitlab
+  renku:
+    version: 'latest'
```

Comment:
I would rather put this in https://github.com/SwissDataScienceCenter/renku/blob/master/charts/renku/values.yaml,
with a proper doc comment

Reply:
My reasoning was that by setting `latest` in `values.yaml`, any deployment would use 'latest' unless it's explicitly overwritten. In the suggested setup it would pick up the chart version from the renku-ui chart by default which I would see as the expected behaviour. But maybe there should be a commented-out hint to the possibility of fixing the renku version in `values.yaml`?

Reply:
The idea is to put `0.2.0` in the default values for the chart with the same tag, and `latest` otherwise. And doing this in the default values, not just minikube values

Reply:
The other reason to put it in `values.yaml` is to document what it does -- not at all obvious if it's hidden in `minikube-values`

Reply:
@rokroskar I agree that indicating this possibility in `values.yaml` is a good idea.

But don't you think that hard-coding this value to '0.2.0' in `values.yaml` is very error prone? I see us forgetting to manually update this value in the future when we have a new release...

Reply:
well that's part of the release. when we are in post-0.2.0 mode we can set it to latest but for the 0.2.0 release it needs to be pinned. Either way it should be in `values.yaml` and documented.

Reply:
I don't want it to be latest! I want it to pick up the version of the chart unless explicitly set to latest for dev purposes, therefore minikube!

Reply:
Ok! I get it! I misunderstood! Is it templated then???? :)

Reply:
The following env var for the ui is set in `deployment.yaml`
```
            - name: RENKU_VERSION
               value: {{ .Values.global.renku.version | default ( .Chart.Version ) | quote }}
```

Reply:
ok but still add it to `values.yaml` just commented out and with a description of what it does.

Reply:
The issue with this setup is that we need to continue releasing the template along with the rest. I suppose that's fine but we need to automate the process